github/libp2p.yml

```diff
@@ -286,11 +286,14 @@ repositories:
         - tbenbow
     default_branch: main
     delete_branch_on_merge: false
+    files:
+      .github/workflows/stale.yml:
+        content: .github/workflows/stale.yml
     has_discussions: false
     merge_commit_message: PR_TITLE
     merge_commit_title: MERGE_MESSAGE
-    secret_scanning_push_protection: false
-    secret_scanning: false
+    secret_scanning_push_protection: true
+    secret_scanning: true
```

Comment:
Seems unrelated to the PR.

Reply:
It is unrelated. It was automatically committed via "fix rules". It's been resolved separately in https://github.com/libp2p/github-mgmt/pull/180 and now these changes were removed from the PR.